rustls/tests/api.rs

```diff
@@ -3689,6 +3689,11 @@ mod test_quic {
             do_exporter_test(client_config, server_config);
         }
     }
+
+    #[test]
+    fn test_quic_version_can_clone() {
+        let _ = quic::Version::V1.clone();
+    }
```

Comment:
Is this really a risk? `#[derive(Clone)]` seems unlikely to malfunction.

Reply:
We have a few other tests like this, presumably to increase test coverage?

Reply:
Does the coverage checker actually count derived traits? Weird.

Reply:
I removed it and it didn't seem to change coverage, so that seems fine.